guides/v2.2/config-guide/varnish/config-varnish-advanced.md

```diff
@@ -89,16 +89,16 @@ Saint mode is not part of the main Varnish package. It is a separately-versioned
 After you've recompiled, you can install the Saint mode {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}. In general, follow these steps:
 
 <ol>
-<li><p>Obtain the source code from <a href="https://github.com/varnish/varnish-modules">Varnish modules</a> . Clone the git version (master version) since the 0.9.x versions contain a source code error.</p></li>
-
+<li><p>Obtain the source code from [Varnish modules](https://github.com/varnish/varnish-modules) . Clone the git version (master version) since the 0.9.x versions contain a source code error.</p></li>
```

Comment:
You still need to remove the HTML formatting from this section as I showed in my previous comment and example.